src/test/scala/org/broadinstitute/dsde/agora/server/webservice/PermissionIntegrationSpec.scala

```diff
@@ -524,36 +523,36 @@
 //      assert(entity.public.contains(false))
 //    }
 //  }
```

Comment:
These still need to be re-enabled, but they're using `querySingleRoute` which needs conversion.